Library/Koala/Koala.swift

```diff
@@ -438,7 +438,7 @@ public final class Koala {
     self.track(event: "Triggered 1Password")
   }
 
-  // MARK: Discovery Events
+  // MARK: - Discovery Events
```

Comment:
It was almost impossible to navigate `Koala.swift`'s jump bar in Xcode so I've decided to jump the gun and update our pragma marks. This adds a separator between each section.